gammapy/modeling/models/tests/test_cube.py

```diff
@@ -783,3 +784,35 @@ def test_spatial_model_io_background(background):
     assert "spatial" in model_dict
     new_model = FoVBackgroundModel.from_dict(model_dict)
     assert isinstance(new_model.spatial_model, ConstantSpatialModel)
+
+
+def test_piecewise_spatial_model_background(background):
+
+    geom = background.geom
+    coords = geom.get_coord().flat
+
+    spatial_model = PiecewiseNormSpatialModel(coords, frame="galactic")
+    reference_npred = TemplateNPredModel(background, spatial_model=None).evaluate()
+    identical_npred = TemplateNPredModel(
```

Comment:
You might use a `FoVBackgroundModel` here no?

Reply:
the equivalent test for the `FoVBackgroundModel` is done below.